cmd/influxd/run/server_test.go

```diff
@@ -1394,6 +1394,11 @@ func TestServer_Query_Common(t *testing.T) {
 			exp:     fmt.Sprintf(`{"results":[{"series":[{"name":"cpu","columns":["time","value"],"values":[["%s",1]]}]}]}`, now.Format(time.RFC3339Nano)),
 		},
 		&Query{
+			name:    "explicitly selecting time and a valid measurement and field should succeed",
+			command: `SELECT time,value FROM db0.rp0.cpu`,
```

Comment:
A test with `select value, time` would be nice as it would show that `time` will always be the first column, but all other columns still maintain their sort order (or should)